benchmarks/314.hadoop/hadoop.sh

```diff
@@ -62,28 +31,35 @@ if [ $? -ne 0 ]; then
     cat ~/.ssh/*.pub >>${f0}
 fi
 
-############## get JAVA_HOME which need to be used later ################
-#Get the java path
-java_loc=$(find /usr/lib/jvm -name 'java-*-openjdk*' |sed -n "1p")
+##############################
+#tmp log dir
+drTmpHadoop=/mnt/tmpLogCaliper/hadoop
+drTmpHadoop=/mnt/hadoop
+grep -q "${drTmpHadoop}" <<< "${HADOOP_TMP}"
+if [ $? -ne 0 ]; then
+    export HADOOP_TMP=${drTmpHadoop}
+fi
+rm -rf "${drTmpHadoop}"
+mkdir -p "${drTmpHadoop}"
+
+##############################
+#JAVA_HOME
+java_loc=$(find /usr/lib -name 'java-*-openjdk*' |sed -n "1p")
```

Comment:
Bel this way of locating java causes problem on CentOS.
Thats why we changed it to /usr/lib/jvm  but not /usr/lib/.

If you want to confirm test this on CentOS.